### ISSUE TITLE
the ansible playbooks will fail the fresh installation of rackhd while using an Ubuntu system i386.

### ISSUE THREAD

**Issue by miramm** [RackHD/RackHD#462]
this failures occur with the Ubuntu Package Based Installation from the RackHD documentation. The steps that fails is te ansible playbooks step. 

root@onrack-virtual-machine:/home/onrack/RackHD/packer/ansible# ansible-playbook -c local -i "local," rackhd_package.yml

PLAY [RackHD Demonstration Server] ******************************************** 

GATHERING FACTS *************************************************************** 
ok: [local]

TASK: [apt | Update apt] ****************************************************** 
ok: [local]

TASK: [build | Install Build Tools] ******************************************* 
changed: [local] => (item=build-essential,genisoimage,debhelper,devscripts,fakeroot)

TASK: [git | Install Git] ***************************************************** 
ok: [local] => (item=git)

TASK: [rabbitmq | Install RabbitMQ] ******************************************* 
changed: [local] => (item=rabbitmq-server)

TASK: [mongodb | Install MongoDB] ********************************************* 
changed: [local] => (item=mongodb)

TASK: [node | Setup nodejs source for installing node v4] ********************* 
changed: [local]

TASK: [node | Install node v4] ************************************************ 
ok: [local] => (item=nodejs)

TASK: [rackhd-packages | shell apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 379CE192D401AB61] *** 
changed: [local]

TASK: [rackhd-packages | apt pkg=python-pycurl state=present] ***************** 
changed: [local]

TASK: [rackhd-packages | apt_repository repo="deb https://dl.bintray.com/rackhd/debian trusty main" state=present] *** 
changed: [local]

TASK: [rackhd-packages | Install RackHD Packages] ***************************** 
failed: [local] => (item=on-dhcp-proxy,on-http,on-taskgraph,on-tftp,on-syslog) => {"failed": true, "item": "on-dhcp-proxy,on-http,on-taskgraph,on-tftp,on-syslog"}
msg: No package matching 'on-dhcp-proxy' is available

FATAL: all hosts have already failed -- aborting

PLAY RECAP ******************************************************************** 
           to retry, use: --limit @/root/rackhd_package.retry

local                      : ok=11   changed=7    unreachable=0    failed=1   

root@onrack-virtual-machine:/home/onrack/RackHD/packer/ansible#

**Comment by panpan0000** [comment]
Hi, @miramm :
as you mentioned your Ubuntu is i386 32bit , as RackHD is designed for enterprise server usage, so all of our debian package are tagged as amd64 architecture . 
I believe that's the reason why any of the on-dhcp-proxy or on-http...etc can't be found at your side.

although RackHD MAY run on 32bit OS, but nobody tried :-)

so the recommendation is to install RackHD on a Ubuntu 14.04 64 bit VM instead.



PS:
@RackHD/corecommitters @yyscamper @anhou @leoyjchang  , Do you think we need a chapter "System Requirement" in readthedoc ?

**Comment by miramm** [comment]
Hi @panpan0000 ,
I did install rackhd into an amd64 and it worked fine. I though it would be good to have the option for i386 as this is open source and not just enterprise servers. in any case, it should be documented to know the minimum requirement. 
thanks.

**Comment by panpan0000** [comment]
@heckj    @RackHD/corecommitters , Can you consider @miramm's idea ?
at least document somewhere ?

**Comment by leoyjchang** [comment]
Yeah, creating "System Requirement" chapter to document minimum requirement sounds reasonable to me. I created PT story to track the document update (https://www.pivotaltracker.com/story/show/133163365) if no one objects.
